torch/nn/modules/normalization.py

```diff
@@ -80,7 +80,7 @@ def extra_repr(self) -> str:
         return '{size}, alpha={alpha}, beta={beta}, k={k}'.format(**self.__dict__)
 
 
-_shape_t = Union[int, List[int], Size]
+_shape_t = Union[int, Sequence[int], Size]
```

Comment:
do we have a test to validate JIT-ability? since we knew Sequence & Tuple doesnt work on JIT.

Reply:
Actually this is not necessary. I am reverting this particular change